docs/src/developing/test-validator.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Solana Test Validator
+---
+During early stage development, it is often convenient to target a cluster with
+fewer restrictions and more configuration options than the public offerings
+provide. This is easily achieved with the `solana-test-validator` binary, which
+starts a local, full-featured, single-node cluster on the developer's workstation.
+
+## Advantages
+- No RPC rate-limits
+- No airdrop limits
+- Direct [on-chain program](on-chain-programs/overview) deployment
+(`--bpl-program ...`)
+- Clone account's from a public cluster, including programs (`--clone ...`)
+- Configurable transaction history (`--limit-ledger-size ...`)
```

Comment:
```suggestion
- Configurable amount of transaction history retained (`--limit-ledger-size ...`)
```
^ I'm not completely happy with this suggestion, but the original text suggests a lot more interesting control than what actually exists